test/controllers/articles_controller_test.rb

```diff
@@ -0,0 +1,48 @@
+require 'test_helper'
+
+class ArticlesControllerTest < ActionDispatch::IntegrationTest
+  setup do
+    @article = articles(:one)
+  end
+
+  test 'should get index' do
+    get articles_url
+    assert_response :success
+  end
+
+  test 'should get new' do
```

Comment:
- [ ] It would be better if we follow the project guidelines by creating a request spec file instead as we have in the screenshot below. So we will have to create a directory called `request` under the `spec` directory. And in the request directory, we create our test files 
![Screenshot from 2023-05-16 02-18-55](https://github.com/djo1975/Bloger/assets/46388113/f297a782-9635-4075-aac5-8b4ba7707b2e)
![Screenshot from 2023-05-16 02-23-44](https://github.com/djo1975/Bloger/assets/46388113/c1ccc087-4f0f-4518-a21d-5cbd64971e19)